crates/typst-library/src/layout/page.rs

```diff
@@ -109,28 +103,24 @@ pub struct PageElem {
     #[ghost]
     pub flipped: bool,
 
-    /// The page's margins.
-    ///
-    /// - `{auto}`: The margins are set automatically to 2.5/21 times the smaller
-    ///   dimension of the page. This results in 2.5cm margins for an A4 page.
-    /// - A single length: The same margin on all sides.
-    /// - A dictionary: With a dictionary, the margins can be set individually.
-    ///   The dictionary can contain the following keys in order of precedence:
-    ///   - `top`: The top margin.
-    ///   - `right`: The right margin.
-    ///   - `bottom`: The bottom margin.
-    ///   - `left`: The left margin.
-    ///   - `inside`: The margin at the inner side of the page (where the
-    ///     [binding]($page.binding) is).
-    ///   - `outside`: The margin at the outer side of the page (opposite to the
-    ///     [binding]($page.binding)).
-    ///   - `x`: The horizontal margins.
-    ///   - `y`: The vertical margins.
-    ///   - `rest`: The margins on all sides except those for which the
-    ///     dictionary explicitly sets a size.
-    ///
-    /// The values for `left` and `right` are mutually exclusive with
-    /// the values for `inside` and `outside`.
+    /// ページの余白。
+    ///
+    /// - `{auto}`: 余白が自動的に短辺の2.5/21倍の大きさに設定されます。
+    /// A4のページでは2.5cmの余白になります。
+    /// - 単一の長さ指定: 全ての辺に同じ大きさの余白を適用します。
+    /// - 辞書指定: 辞書を用いることで余白を個々に設定可能です。
+    ///  この辞書に含められるキーは以下のとおりです。リストは優先順位の順に並んでいます。
+    ///   - `top`: 上部の余白。
+    ///   - `right`: 右側の余白。
+    ///   - `bottom`: 下部の余白。
+    ///   - `left`: 左側の余白。
+    ///   - `inside`: ページの内側の余白([綴じ]($page.binding)側).
+    ///   - `outside`: ページの外側の余白([綴じ]($page.binding)の反対側).
+    ///   - `x`: 水平方向の余白。
+    ///   - `y`: 垂直方向の余白。
+    ///   - `rest`: 明示的に指定されていない残り全ての余白。
+    ///
+    /// `left`と`right`の値は`inside`と`outside`の値と相互排他です。
```

Comment:
「相互排他」は一般的な言葉で意味も通じると思うのですが、補足がある方が読者に優しいかもしれません。（原文から追加していますし、人によっては冗長に感じるかも？）

```suggestion
    /// `left`と`right`の値は`inside`と`outside`の値と相互排他で、同時には指定できません。
```

Reply:
もう少し簡単にできると思います。
```suggestion
    /// `left`と`right`の値は`inside`と`outside`の値と同時に指定できません。
```